src/app/features/admin-home/components/admin-registration-version/admin-registration-version.component.ts

```diff
@@ -1,8 +1,31 @@
-import { Component } from "@angular/core";
```

Comment:
Unit tests for the changes ?

Reply:
Th unit test would be covered in the https://github.com/nuxeo/nuxeo-admin-console-ui/pull/20/files in a separate Pr